source/authentication/email-password.txt

```diff
@@ -93,6 +94,22 @@ Configuration
 
 User Confirmation
 -----------------
+User confirmation occurs when a user wants to create a new Email/password 
+account. Atlas App Services supports three methods for confirming a user account:
+
+- Send a confirmation email
+- Run a confirmation function
+- Automatically create the account with no validation.
```

Comment:
For consistency, we should either use punctuation for all or none of these bullet points.

```suggestion
- Automatically create the account with no validation
```